### PR TITLE
add AUFSWhiteouts to convert Overlay whiteouts to AUFS

### PR DESCRIPTION
**Pull request by unclejack**
This PR is ready.

### CODE REVIEW

**Comment by erikh** [comment]
can we put some tests in?

File: headers.json [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[
```
Comment (erikh) [inline headers.json]: can you please either directly indent or run marshal indent this struct so it's easier to edit later?

**Comment by erikh** [comment]
LGTM minus the indent comment

**Comment by unclejack** [comment]
@erikh: I've formatted headers.json. PTAL

**Pull request merged**